src/components/author/_author.scss

```diff
@@ -2,33 +2,30 @@
   align-items: center;
   display: flex;
 
-  &--margin-top {
-    margin-top: $bio-spacing-md;
-  }
-
   &__media {
     display: block;
 
     img {
       border-radius: $border-radius-full;
       display: block;
-      height: $size-large;
-      width: $size-large;
+      height: 2.5rem;
```

Comment:
Why taking these out of a variable, where they will always be the same value, it'll now take 2 updates to change the value, rather than 1